tests/functional/conftest.py

```diff
@@ -1,4 +1,12 @@
 import pytest
+import yaml
```

Comment:
Picky comment, is isort being used here? The imports don't seem to be in alphabetical order